PittsburghServices/pittsburgh_services_scraper.py

```diff
@@ -80,19 +80,14 @@ def grab_data(self):
     service_summary="",
     check_collection="services",
     dump_collection="tmpPittsburghServices",
-    dupe_collection="tmpPittsburghServicesFoundDuplicates",
-    data_source_collection_name="pittsburgh_services",
-    collection_dupe_field='name'
+    dupe_collection="tmpPittsburghServicesDuplicates",
+    data_source_collection_name=data_source_name,
+    collection_dupe_field='name',
+    groupby_columns=['address']
```

Comment:
name should atleast be there isn't it.

Reply:
This one, after checking again on the dataset, I think address is a bit better, because from this data we only want address, city, state, zip and serviceSummary, so we are not losing any data. If we wanted to get the schedule then we should groupby name instead of address, but right now is almost the same.